conf/configuration.go

```diff
@@ -0,0 +1,88 @@
+package conf
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+var (
+	// configuration is the current state of the configuration.
+	configuration = Config{
```

Comment:
f/u 

I think there was a misunderstanding. It's fine for this to be exposed as `DefaultConfiguration` for testing or whatever.
But we should never assign to global vars like we do in `LoadConf`.

Right now we got an unnecessary race condition in `LoadConf` when we call it from 2 threads because both are assigning to a global var. During a test which spins up 2 pinners this will fail with a race condition.

So we don't need `func Conf() Config`. Just a `func LoadConf() (Config, error) {` which creates a new `Conf` instance. That way we can call it a million times in parallel without the race detector going off because we are assigning to the same var.

Reply:
https://github.com/SkynetLabs/pinner/issues/6